ui/lib/replication/addon/templates/components/replication-summary.hbs

```diff
@@ -342,17 +342,43 @@
   {{#if (eq replicationAttrs.mode 'initializing')}}
     The cluster is initializing replication. This may take some time.
   {{else}}
-    {{info-table-row label="Mode" value=replicationAttrs.mode}}
-    {{info-table-row label="Replication set" value=replicationAttrs.clusterId}}
-    {{info-table-row label="Secondary ID" value=replicationAttrs.secondaryId}}
-    {{info-table-row label="State" value=replicationAttrs.state}}
-    {{info-table-row label="Primary cluster address" value=replicationAttrs.primaryClusterAddr}}
-    {{info-table-row label="Replication state" value=replicationAttrs.replicationState}}
-    {{info-table-row label="Last WAL entry" value=replicationAttrs.lastWAL}}
-    {{info-table-row label="Last Remote WAL entry" value=replicationAttrs.lastRemoteWAL}}
-    {{info-table-row label="Merkle root index" value=replicationAttrs.merkleRoot}}
-    {{#if replicationAttrs.syncProgress}}
-      {{info-table-row label="Sync progress" value=(concat replicationAttrs.syncProgress.progress '/' replicationAttrs.syncProgress.total)}}
-    {{/if}}
+    <div class="replication">
+      <div class="selectable-card-container">
+        {{!-- TODO: there are 2 attributes for both state and lastWAL. which one do i use? --}}
+        <ReplicationPrimaryCard
+          @title='State'
+          @description='Updated every ten seconds.'
+          @metric={{replicationAttrs.state}} />
+        <ReplicationPrimaryCard
+          @title='Last WAL entry'
+          @description='Index of last Write Ahead Logs entry written on local storage.'
+          @metric={{replicationAttrs.lastWAL}}
+          />
+
+        <div class="selectable-card is-rounded secondaries">
+          <div class="level">
+          <code class="card-title title is-5">Known Secondaries</code>
+            <ToolbarLink @params={{array "vault.cluster.settings.auth.configure" model.id}} @data-test-manage-link="true">
+              Manage
+            </ToolbarLink>
+          </div>
+          <div>
+            <ReplicationTable />
+            {{!-- TODO: make sure this works --}}
+            {{!-- {{info-table-row label="Secondary ID" value=replicationAttrs.secondaryId}} --}}
+          </div>
+        </div>
+      </div>
+      <div>
```

Comment:
After we merge this in, we can look at using the contextual replication-table component.   Or we can do it in this PR if you want.